agent/virus_total_agent.py

```diff
@@ -106,10 +109,19 @@ def _prepare_targets(self, message: msg.Message) -> list[str]:
         from the config."""
         if message.data.get("host") is not None:
             host = str(message.data.get("host"))
-            if message.data.get("mask") is None:
+            mask = message.data.get("mask")
+            if mask is None:
                 ip_network = ipaddress.ip_network(host)
             else:
-                mask = message.data.get("mask")
+                version = message.data.get("version")
+                if version == 4 and int(mask) < IPV4_CIDR_LIMIT:
+                    raise ValueError(
+                        f"Subnet mask below {IPV4_CIDR_LIMIT} is not supported."
+                    )
+                if version == 6 and int(mask) < IPV6_CIDR_LIMIT:
+                    raise ValueError(
+                        f"Subnet mask below {IPV6_CIDR_LIMIT} is not supported."
+                    )
```

Comment:
This is by-passable, if we send a message with hard-coded version value different than 4 or 6.

Reply:
@deadly-panda  can we set the version manually from ostorlab oss ?

Reply:
@benyissa is it same for other agents ?

Reply:
@elyousfi5 We did that only in a tsunami.